lib/Db/Provider.php

```diff
@@ -36,6 +36,7 @@
  * @method void setClientSecret(string $clientSecret)
  * @method string getDiscoveryEndpoint()
  * @method void setDiscoveryEndpoint(string $discoveryEndpoint)
+ * @method void setEndSessionEndpoint(string $endSessionEndpoint)
```

Comment:
We need `getEndSessionEndpoint` as well.

Reply:
Added now - thanks for the suggestion! :)